pcsx2/gui/AppMain.cpp

```diff
@@ -542,11 +542,13 @@ void Pcsx2App::LogicalVsync()
 	FpsManager.DoFrame();
 	
 	if (EmuConfig.Gamefixes.FMVinSoftwareHack || g_Conf->GSWindow.IsToggleAspectRatioSwitch) {
+		AspectRatioType& art = g_Conf->GSWindow.AspectRatio;
+		if (g_Conf->GSWindow.IsToggleAspectRatioSwitch && !EnableFMV && !FMVstarted && art == AspectRatio_4_3 && prevAR != art)
+			prevAR = art;
```

Comment:
It's needed for when you manually switch to 4:3 aspect ratio after using the widescreen option before. For example when switching games without shutting down PCSX2.